config/samples/atlas_v1_atlasproject_with_maintenance_window.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: atlas.mongodb.com/v1
+kind: AtlasProject
+metadata:
+  name: my-project
+spec:
+  name: Test Atlas Operator Project
+  projectIpAccessList:
+    - ipAddress: "192.0.2.15"
+      comment: "IP address for Application Server A"
+  maintenanceWindow:
+    dayOfWeek: 3
+    hourOfDay: 5
+    autoDefer: true
+    startASAP: false
```

Comment:
Do we really need to define all of the fields here? I think it's better to omit optional fields with default values.